### PR TITLE
Improve design

### PR DESCRIPTION
**Pull request by marialungu**

### CODE REVIEW

File: Backend/admin/src/screens/news/news.css [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
-    flex-wrap: wrap;
     flex: 1 1 auto;
-    justify-content: center;
+    justify-content: space-between;
     align-items: center;
 }
 
 .margin{
-    width: 200px;
+    width: 470px;
```
Comment (RafaelClaudiu) [inline Backend/admin/src/screens/news/news.css]: Fix width ? Won t be better to have a min and a max ? Though the card is small enough.
But still the css class is named margin

File: Backend/admin/src/screens/messages/messages.js [hunk 2]
```diff
@@ -42,77 +41,63 @@ const Messages = () => {
...
+      title="Mesaje"
+      extra={
+        <Button type="primary" shape="round" onClick={handleCreateMessage}>
+          Adauga un mesaj
+        </Button>
+      }
+    >
+      <List
+        itemLayout="vertical"
+        dataSource={messages}
```
Comment (RafaelClaudiu) [inline Backend/admin/src/screens/messages/messages.js:55]: Another thing if you have time and if you want.
Try and think what happens if the retrieving the messages will take around 30 seconds. For that thirty seconds you will see no data, which is not true. Can you add will retrieving the data to render skeleton with animating on true ? 

Nice to have the skeleton in the list or the content design

File: Backend/admin/src/screens/messages/message-form.js [hunk 3]
```diff
@@ -1,33 +1,30 @@
...
   };
 
-  return(
-    <div className="message-form-wrapper">
-      <div className="message-form">
+  return (
+    <div className="form-wrapper">
+      <Card title="Adauga un mesaj">
         <Form
           initialValues={values}
```
Comment (RafaelClaudiu) [inline Backend/admin/src/screens/messages/message-form.js:24]: Same here on edit. If I am waiting 2 mins for the message to be retrieved ? You are rendering null which means I don't know if the request is in pending. 
Can you also add here the skeleton with the animation , this is letting the user know that some request is pending.

File: Backend/admin/src/screens/advices/advice-form.js [hunk 4]
```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Button, Form, Input, DatePicker } from 'antd';
+import { Button, Form, Input, DatePicker, Row, Card } from 'antd';
 import moment from 'moment';
-import './advice-form.css';
+import '../styles.css';
+import { useHistory } from 'react-router';
```
Comment (bnenu) [inline Backend/admin/src/screens/advices/advice-form.js:5]: Cred ca aici e un lint issue  - import order

File: Backend/admin/src/screens/advices/advices.js [hunk 5]
```diff
@@ -42,26 +44,24 @@ const Advices = () => {
...
-          </>
-        }
-        itemLayout="horizontal"
+        itemLayout="vertical"
         dataSource={advices}
         renderItem={({ body, creationDate, publishDate, id }) => (
           <List.Item
+            className="general-wrapper"
             actions={[
               // eslint-disable-next-line jsx-a11y/anchor-is-valid
```
Comment (bnenu) [inline Backend/admin/src/screens/advices/advices.js:62]: I like that!!

**Review by bnenu — APPROVED** [review]

**Review by RafaelClaudiu — APPROVED** [review]

**Pull request merged**